### PR TITLE
LPS-179505 Unexpected "Description" subtitle when creating a web content

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPS-179505](https://issues.liferay.com/browse/LPS-179505)

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:data-engine

File: modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/DefaultVariant.es.js [hunk 1]
```diff
@@ -145,15 +145,8 @@ export function Page({
...
-	return (
-		<>
-			{title && <h2 className="lfr-ddm-form-page-title">{title}</h2>}
-			{description && (
-				<h3 className="lfr-ddm-form-page-description">{description}</h3>
-			)}
-		</>
-	);
+export function PageHeader({title}) {
+	return <>{title && <h2 className="lfr-ddm-form-page-title">{title}</h2>}</>;
```
Comment (alinedoleron) [inline modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/DefaultVariant.es.js:149]: Hello, @larissacribeiro, I've seen that removing the `description` here, it also removes it from the **published Form pages**. Just to make sure, this ticket is just related to Web Content, right? If so, you should make some restriction just to Web Content.
@carolmariaabb, could you confirm if we should keep the page descriptions on **Forms**? 

Thanks!
image

Comment (carolmariaabb) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/DefaultVariant.es.js:149]: @alinedoleron, yes we should keep the page descriptions on Forms

Comment (larissacribeiro) [reply modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/DefaultVariant.es.js:149]: Hi @alinedoleron, I see, gonna work on it now!

**Comment by carolmariaabb** [comment]
@larissacribeiro I'm going to close this PR for now, but feel free to reopen it or even open a new one when needed.
Thanks!

**Pull request closed without merging**